Rakefile

```diff
@@ -27,6 +27,16 @@ RSpec::Core::RakeTask.new(:acceptance) do |t|
     t.pattern = 'spec/acceptance'
 end
 
+desc "Run integration tests"
+RSpec::Core::RakeTask.new(:integration) do |t|
```

Comment:
The `Gemfile` has `:system_tests` and here we have `:integration`.  Would be good to pick one nomenclature.